src/libs/monitoring/scrubbing.ts

```diff
@@ -30,7 +30,7 @@ export const scrubPhoneNumbers = (
     if (hasConsoleArguments(event)) {
       const message = event.data.arguments[0]
 
-      if (message?.includes('tel:')) {
+      if (message.includes('tel:')) {
```

Comment:
I think it was an eslint --fix modification, I didn't see it, good catch. I'm with you on that so I guess I'm gonna tell typescript it can in fact be undefined